projects/schematics/src/add-ssr/index.ts

```diff
@@ -42,11 +42,16 @@ import {
   addToModuleImportsAndCommitChanges,
 } from '../shared/utils/module-file-utils';
 import {
-  getAngularVersion,
   getSpartacusSchematicsVersion,
   readPackageJson,
 } from '../shared/utils/package-utils';
 
+const DEPENDENCY_NAMES: string[] = [
+  '@angular/platform-server',
+  NGUNIVERSAL_EXPRESS_ENGINE,
+  'ts-loader',
```

Comment:
So it's really needed?

Reply:
Not sure. It does compile without it, but I'd rather have @dunqan double check it.

Reply:
And it's not really in the scope of this ticket, I'd say :)